### PR TITLE
July update enrich business error message

### PR DESCRIPTION
**Pull request by meganelepalud-akeneo**

### CODE REVIEW

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+We improved the connection error messages by adding information that will help you to fix them. In the `Monitoring` page, you will now find links to other PIM page or help center pages, and some data about the related product.
```
Comment (NolwennP) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:13]:
```suggestion
We improved the connection error messages by adding information that will help you to fix them. In the `Monitoring` page, you will now find links to other PIM pages or help center pages, and some data about the related product.  
```

Comment (NolwennP) [reply content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:13]: What do you mean by "related product"?

Comment (meganelepalud-akeneo) [reply content/updates/2020-07/1-easily-fix-errors-on-your-connections.md]: We only monitor product errors so there is always a product related to an error message

File: content/updates/2020-07/1-easily-fix-errors-on-your-connections.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+pim_announcement_audience:
+- EE
+- CE
+---
+
+# Easily fix errors on your connections 
+::: meta-data type="Improvement" features="Connections, Productivity" available="early July" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
+:::
+
+We improved the connection error messages by adding information that will help you to fix them. In the `Monitoring` page, you will now find links to other PIM pages or help center pages, and some data about the related product.
```
Comment (LaureBro) [inline content/updates/2020-07/1-easily-fix-errors-on-your-connections.md:13]:
```suggestion
To facilitate connection management, we have improved the connection warning message visible under the `Monitoring` page. Each warning message shows a small text that explains what went wrong during the connection's synchronization, internal links to help you check the data in your PIM, and external links to directly access useful information in our help center.
```

**Pull request merged**